### ISSUE TITLE
Not compitible for firefox 57.x

### ISSUE THREAD

**Issue by ishah09** [Noitidart/NativeShot#78]
having firefox 57.0b8 (64-bit) version and native shot is not working for same

**Comment by Noitidart** [comment]
Yes it will not work, Firefox changed too much. Thanks for being a user! This is a piece of software I use everyday so I really have motivation to finish the desktop client over at - https://github.com/Noitidart/NativeShotDesktop

**Comment by ishah09** [comment]
Yes I do use this extension from last 9 months and it works great overall. I must recommend it. Please get it resolve asap and make it compatible for Firefox new version.

**Comment by ishah09** [comment]
@Noitidart, 
Any update on this..? 
When plugin is giving next update..? Awaiting for Firefox-57

**Comment by renatofrota** [comment]
Hello @Noitidart ,

Do you know any other FF extension capable of taking fullscreen screenshots (capturing other windows outside FF and the window itself) as NativeShot is/were?

Thanks in advance
